test/org/traccar/protocol/AplicomProtocolDecoderTest.java

```diff
@@ -70,6 +71,9 @@ public void testDecode() throws Exception {
         verifyPosition(decoder, binary(
                 "44C20146B710C158DA009500B09F7700C054CA0EA454CA0EA403BE0BF6015D706B070000142A600000000000000002434946010801000754CA0EA4000000000000008400000000000000000000000000000000300000FE00FE0000000000000000000000000000000000000000000000000000000000000000000040502035000000000000020D0000030D0000040C0000040D0000050C0000050D0000058C0000060C"));
 
+        verifyPosition(decoder, frameDecoder.decode(null, null,
+                binary("33353733303030373030393233333644C20146B710C158DA009500B09F7700C054CA0EA454CA0EA403BE0BF6015D706B070000142A600000000000000002434946010801000754CA0EA4000000000000008400000000000000000000000000000000300000FE00FE0000000000000000000000000000000000000000000000000000000000000000000040502035000000000000020D0000030D0000040C0000040D0000050C0000050D0000058C0000060C")));

```

Comment:
This is not a valid frame. You should not test fame decoder in protocol decoder test class.